api/v1/_submissions/BackendSubmissionsHandler.inc.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file api/v1/_submissions/BackendSubmissionsHandler.inc.php
```

Comment:
I'd be fine with `/backend/submissions`, as long as that endpoint get it's own handler. I just don't want all backend endpoints in the same endpoint handler.

I had trouble getting any API handler registered on anything other than `/api/v1/<endpoint>`. I couldn't seem to get `/api/v1/backend/<endpoint>` working. I then thought of the `_` approach which resolved my issue. But happy to implement the `/backend/submissions` approach if you can show me how to properly route requests there.